_topic_map.yml

```diff
@@ -517,6 +517,8 @@ Topics:
 - Name: Understanding the Ingress Operator
   File: ingress-operator
   Distros: openshift-enterprise,openshift-webscale,openshift-origin
+- Name: Configuring node port service range
```

Comment:
Same question about whether there should be a "the" in here.

Reply:
I'll buy an article for this.